### ISSUE TITLE
Angular Project Template does not work with `ng` commands

### ISSUE THREAD

**Issue by rhires** [aspnet/Docs#5927]
## Issues with Existing Topic

https://docs.microsoft.com/en-us/aspnet/core/spa/angular?tabs=netcore-cli

This comment is as of April 7, 2018, and pretty much spells out the issue: 

Well... This is no good... Installed .NET Core SDK 2.1. 

Ran "dotnet new angular -o myproject"

went to the newly created directory and ran NPM install.

*Everything is great... ran the application and it loads on my web browser

Then went to the ClientApp directory and ran "ng g c mycomponent"

and got this error: "Unable to find any apps in `.angular-cli.json`"

Looked at the directory and noticed that there is no .angular-cli.json file at the root of the ClientApp directory.

Tried it twice BTW, and same results. I know I can go an create the file, but I'm pretty new to angular, and now I got to break my head because the template is not complete.

So basically, there is something missing from the template install.

**Comment by scottaddie** [comment]
Moved issue to https://github.com/aspnet/templating/issues/473 for review.
